packages/styles/src/core/containers/test/containers-in-sidebar-bottom-and-legends.html

```diff
@@ -25,7 +25,7 @@
           <div class="as-panel as-panel--top as-panel--right">
             <div class="as-panel__element">
               <section class="as-box">
-                <p class="as-title">LEGEND 1a</p>
+                <p class="as-title">LEGENDICA 1a</p>
```

Comment:
👀

Reply:
It's was a trap to check if you were paying attention.